pkg/globalnet/controllers/ipam/gatewaymonitor_test.go

```diff
@@ -333,7 +333,7 @@ var _ = Describe("Node monitoring", func() {
 		node = &corev1.Node{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:        nodeName,
-				Annotations: map[string]string{constants.CniInterfaceIP: "10.20.30.40"},
+				Annotations: map[string]string{constants.CNIInterfaceIP: "10.20.30.40"},
```

Comment:
CNInterface?? But CNIInterface is more reading-friendly. I don't have strong opinion about this.